src/search/utils/QueryStringGenerator.ts

```diff
@@ -51,17 +53,20 @@ const createValueString = (
     throw new Error('term is undefined');
   }
   if (id) {
-    return `"${id}"`;
+    // We need to not quote ec numbers because this will cause the
```

Comment:
If there are wildcards it means we don't have an id as it wasn't selected from suggester? If so this is unnecessary.

Reply:
fixed